soccer/gameplay/robocup-py.cpp

```diff
@@ -8,6 +8,8 @@
 #include <string>
 #include <utility>
 
+#include "WorldState.hpp"
```

Comment:
Changes in `robocup-py.cpp` are essentially trivial and just relate to the ball interface changes.